src/media/mod.rs

```diff
@@ -208,25 +217,24 @@ impl ::std::fmt::Display for MediaCategory {
     }
 }
 
-/// A builder struct that allows you to build up parameters to a media upload before initiating it.
+/// A builder that provides mechanism to create upload and customize it.
```

Comment:
Looking at other builders i've done before, they seem to start differently than this anyway, so how about:

> Represents a media upload before it is sent.

This matches `DraftTweet`, as well as the search builders in `place` and `search`.

Reply:
Sounds good, having similar style for all builders is good